modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -64,6 +64,9 @@ These are some things you need to keep in mind when creating your input files:
 ====
 Indexes and constraints are not created during the import.
 Instead, you have to add these afterward (see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/indexes-for-full-text-search[Cypher Manual -> Indexes]).
+
+Starting from Neo4j 5.24, you can use the `--schema` option to create indexes and contraints during the import process.
```

Comment:
@docrozza, is this option for the community edition?

Reply:
It only works for block format - so kinda enterprise by default
Also, only FULL at the moment but it will be available for incremental at some point (still enterprise though)